config/mocha.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  plugins: [
+    'mocha',
+  ],
+
+  env: {
+    'mocha': true,
+  },
+
+  rules: {
+    'mocha/handle-done-callback': 'error',
+    'mocha/max-top-level-suites': [ 'error', { limit: 1 } ],
+    'mocha/no-async-describe': 'error',
+    'mocha/no-exclusive-tests': 'error',
+    'mocha/no-global-tests': 'error',
+    'mocha/no-hooks': 'error',
```

Comment:
We make _extensive_ use of hooks in the extension. This seems like it'd require a massive effort to change. Even in your draft extension PR, hooks are used all over the place.

Reply:
I see that you've disabled this rule in the extension config. I would suggest that we'd might as well enable it here as well, because going forward we shouldn't introduce Mocha to any new projects anyway (we should use Jest instead). So this shared config should represent our "legacy" config, as is currently used in the extension. Unless you want to change it in the extension, as you did with the arrow function rule.

Plus... hooks are fine 🤷‍♂. It's easy to write bad tests with hooks, but they're not _all_ bad.

Reply:
Sure, I'll disable it here as well.